src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx

```diff
@@ -1,20 +1,59 @@
 import { Box, BoxProps, Input } from "@artsy/palette"
 import { useFormikContext } from "formik"
+import { useEffect } from "react"
+import { useRouter } from "v2/System/Router/useRouter"
+import { ContactInformation_me } from "v2/__generated__/ContactInformation_me.graphql"
+import { useSubmission } from "../../Utils/useSubmission"
 
+export const getContactInformationFormInitialValues = (
+  me: ContactInformation_me
+) => ({
+  name: me?.name || "",
+  email: me?.email || "",
+  phone: me?.phone || "",
+})
 export interface ContactInformationFormModel {
   name: string
   email: string
   phone: string
 }
 
-export interface ContactInformationFormProps extends BoxProps {}
+export interface ContactInformationFormProps extends BoxProps {
+  me: ContactInformation_me
+}
 
 export const ContactInformationForm: React.FC<ContactInformationFormProps> = ({
+  me,
   ...rest
 }) => {
-  const { values, handleChange, handleBlur } = useFormikContext<
-    ContactInformationFormModel
-  >()
+  const {
+    values,
+    handleChange,
+    handleBlur,
+    resetForm,
+    validateForm,
+    setErrors,
+  } = useFormikContext<ContactInformationFormModel>()
+
+  const {
+    match: {
+      params: { id },
+    },
+  } = useRouter()
+  const { submission } = useSubmission(id)
+
+  useEffect(() => {
+    if (submission) {
+      resetForm({ values: submission.contactInformationForm })
+      validateForm(submission.contactInformationForm).then(e => {
+        setErrors(e)
```

Comment:
it looks like these errors are effective only for enabling/disabling the submit button, but we don't seem to display them in the ui. I'm wondering if there's any reason why we don't display them to the user?

Reply:
requirements)

Reply:
You can check this ticket for example: [SWA-66](https://artsyproduct.atlassian.net/browse/SWA-66)

Reply:
then we can get rid of the chaining `.then(e => { setErrors(e) })` -> because it's creating the impression that errors are handled, although they are not. 

what do you think?

Reply:
I think you were right, fixed